src/main/java/com/twilio/rest/api/v2010/account/Address.java

```diff
@@ -174,7 +175,10 @@ private Address(
         final Boolean validated,
 
         @JsonProperty("verified")
-        final Boolean verified
+        final Boolean verified,
+
+        @JsonProperty("street_secondary")
```

Comment:
This should not be included.

Reply:
reverted the new api-definition changes. Generated the code via current twilio-oai definitions